backend/src/schema/resolvers/statistics.js

```diff
@@ -28,6 +28,11 @@ export default {
           const stat = statistics[mapping[key]]
           response[key] = stat ? stat.toNumber() : 0
         })
+
+        /*
+         * Note: invites count is calculated this way because invitation codes are not in use yet
```

Comment:
We learned the day before yesterday that we open registration for non-donators in two weeks from now. So, Invitation codes will not be necessary anymore. (Just as a side note)

Reply:
Thanks, good to know about invitation codes.